receiver/k8sobjectsreceiver/unstructured_to_logdata.go

```diff
@@ -58,6 +59,13 @@ func unstructuredListToLogData(event *unstructured.UnstructuredList, observedAt
 			if namespace := e.GetNamespace(); namespace != "" {
 				resourceAttrs.PutStr(semconv.AttributeK8SNamespaceName, namespace)
 			}
+
+			if config.gvr.Version == "events.k8s.io/v1" {
```

Comment:
```suggestion
			if config.gvr.Group == "events.k8s.io" {
```

Could avoid the extra fn call for events by checking `config.gvr.Resource == "events"`